sdk/src/Core/Amazon.Runtime/Credentials/ECSTaskCredentials.cs

```diff
@@ -1,131 +0,0 @@
-﻿/*
```

Comment:
For later, but these removals impact PowerShell too (I commented on this class because @afroz429 noticed it was used in https://github.com/aws/aws-tools-for-powershell/blob/fe860a680057dfcbccdb95cafd8dd1a361b3f0e0/modules/AWSPowerShell/Common/CredentialsArguments.cs#L212-L213 - but I wouldn't be surprised if there are others as well).

Reply:
okay, thanks for the callout